_data/projects/raiden.yml

```diff
@@ -1,16 +0,0 @@
----
-name: Raiden Network
-desc: A proposed extension to Ethereum which scales-out asset transfer capacity in the network.
-site: https://github.com/raiden-network/raiden/
-tags:
-- blockchain
-- ethereum
-- payment-channels
-- microtransactions
-- state-channels
-upforgrabs:
-  name: good first issue
-  link: https://github.com/raiden-network/raiden/labels/good%20first%20issue
```

Comment:
I thought we were going to use https://github.com/raiden-network/raiden/labels/Tag%20%2F%20Good%20First%20Issue instead?